data/kahet/beyond patir.txt

```diff
@@ -0,0 +1,2432 @@
+		"lifetime" 40
+		"submunition" "track3 print right"
+		"submunition" "spin4 print left"
+
+outfit "spin3 print left"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"velocity" -25
+		"random velocity" 20
+		"inaccuracy" 5
+		"acceleration" -1
+		"live effect" "asteroid circle trail" 5
+		"drag" .1
+		"turn" 5
+		"lifetime" 40
+		"submunition" "track3 print left"
+		"submunition" "spin4 print right"
+
+outfit "track3 print right"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+		"submunition" "spin4 print right"
+		"submunition" "spin4 print left"
+
+outfit "track3 print left"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+		"submunition" "spin4 print right"
+		"submunition" "spin4 print left"
+
+outfit "spin4 print right"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"velocity" -25
+		"random velocity" 20
+		"inaccuracy" 5
+		"acceleration" -1
+		"live effect" "asteroid circle trail" 5
+		"drag" .1
+		"turn" -5
+		"lifetime" 40
+		"submunition" "track4 print right"
+
+outfit "spin4 print left"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"velocity" -25
+		"random velocity" 20
+		"inaccuracy" 5
+		"acceleration" -1
+		"live effect" "asteroid circle trail" 5
+		"drag" .1
+		"turn" 5
+		"lifetime" 40
+		"submunition" "track4 print left"
+
+outfit "track4 print right"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+
+outfit "track4 print left"
+	weapon
+		sprite "projectile/asteroid weapon swarm"
+			"frame rate" 5
+			"random start frame"
+		"acceleration" 1
+		"drag" .1
+		"turn" 20
+		"live effect" "asteroid circle trail" 5
+		"homing" 3
+		"tracking" 1
+		"lifetime" 550
+		"inaccuracy" 5
+		"split range" 180
+		"submunition" "asteroid laser 1"
+			"offset" 0 10
+
+outfit "asteroid laser 1"
+	weapon
+		sprite "projectile/asteroid laser"
+			"frame rate" 60
+			"random start frame"
+		sound "zapper"
+		"hit effect" "skylance impact"
+		"target effect" "asteroid target"
+		"inaccuracy" .3
+		"velocity" 200
+		"lifetime" 1
+		"reload" 1
+		"firing heat" 1.4
```

Comment:
If this only ever exists as a submunition, then firing heat won't do anything.

Reply:
Removed